alma/alma-woocommerce-gateway
# fix: amount in installment plan checkout block + update translation

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2237/[-bug]-installments-plans-dont-have-the-right-amount-in-block)

### Code changes


Our checkout block doesn't display the right installment plan. We fix it by changing the value in the plan by the total_amount of the installment

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Install a new version of WooCommerce since 8.3.0
Enable the Block option in our module
Display the checkout page with an Alma plan with PnX installment. Check the amount of the installments

### Checklist for authors and reviewers




### Non applicable
